### PR TITLE
Fixes #31643 - Add rubygem-smart_proxy_container_gateway package

### PR DESCRIPTION
**Pull request by ianballou**

### CODE REVIEW

File: comps/comps-foreman-el8.xml [hunk 1]
```diff
@@ -410,8 +410,8 @@
       <packagereq type="default">rubygem-gitlab-sidekiq-fetcher-doc</packagereq>
       <packagereq type="default">rubygem-globalid-doc</packagereq>
       <packagereq type="default">rubygem-google-api-client-doc</packagereq>
-      <packagereq type="default">rubygem-google-cloud-env-doc</packagereq>
       <packagereq type="default">rubygem-googleauth-doc</packagereq>
+      <packagereq type="default">rubygem-google-cloud-env-doc</packagereq>
```
Comment (jlsherrill) [inline comps/comps-foreman-el8.xml:414]: is this change intentional?

Comment (ianballou) [reply comps/comps-foreman-el8.xml:414]: One of the scripts did it, I'll remove the change though

**Comment by ianballou** [comment]
Should be all set now.

**Review by jlsherrill — APPROVED** [review]
